db/schema.rb

```diff
@@ -22,7 +22,7 @@
     t.integer "volume"
     t.integer "issue"
     t.boolean "is_published"
-    t.datetime "created_at", null: false
+    t.datetime "created_at", default: "2017-11-24 10:48:48", null: false
```

Comment:
Why this default?

Reply:
LOL sorry, this was when I was trying to fix the date. I'll push the migration soon.